### PR TITLE
feat(ui): Adds Health project feature flag, menu item, dummy screens

### PR DESCRIPTION
**Pull request by matejminar**
Adds Health project feature flag, menu item, dummy screens.

### CODE REVIEW

File: src/sentry/static/sentry/app/views/health/detail/index.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+    return (
+      <React.Fragment>
+        <GlobalSelectionHeader organization={organization} />
+
+        <NoProjectMessage organization={organization}>
+          <PageContent>
+            <PageHeader>
+              <PageHeading withMargins>
+                {t('Health Detail ')} {this.props.params.healthSlug}
```
Comment (mitsuhiko) [inline src/sentry/static/sentry/app/views/health/detail/index.tsx]: Not that it matters that much but i think the pattern here should generelly be `{t('Foo')}{' '}`. At least we should try to not make it worse than it already is.

File: src/sentry/static/sentry/app/views/health/landing.tsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+    return (
+      <React.Fragment>
+        <GlobalSelectionHeader organization={organization} />
+
+        <NoProjectMessage organization={organization}>
+          <PageContent>
+            <PageHeader>
+              <PageHeading withMargins>
+                {t('Health Detail ')} {this.props.params.healthSlug}
```
Comment (mitsuhiko) [inline src/sentry/static/sentry/app/views/health/landing.tsx]: Same here.

**Review by mitsuhiko — CHANGES_REQUESTED** [review]

File: src/sentry/static/sentry/app/views/health/detail/index.tsx (hunk 1 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/views/health/detail/index.tsx]: I am aware of that. The trailing space in translation was there by accident and then copied to another file. Thanks for noticing, fixed it.

**Review by billyvg — APPROVED** [review]
Here are some other things that I do when creating a new feature, that you may (or may not) want to do:

- Add feature flag to `getsentry` so that we can internally view the feature (or via the sentry-experiments team)
- Add a percy snapshot via acceptance test - I will sometimes use these screenshots to post in the PR, but it also allows reviewers to easily see what changes there are.

**Review by mitsuhiko — APPROVED** [review]

**Comment by matejminar** [comment]
@billyvg 
> Add feature flag to `getsentry` so that we can internally view the feature (or via the sentry-experiments team)

Thanks, I added the health feature to sentry-experiments team: https://github.com/getsentry/getsentry/pull/3458

> Add a percy snapshot via acceptance test - I will sometimes use these screenshots to post in the PR, but it also allows reviewers to easily see what changes there are.

I am preparing a follow-up PR where I will definitely add one

**Pull request merged**